.readthedocs.yml

```diff
@@ -2,5 +2,7 @@ requirements_file: dev_tools/conf/pip-list-dev-tools.txt
 build:
   image: latest
 python:
-  version: 3.6
+  version: 3.7
   pip_install: true
+  extras_requirements:
```

Comment:
This should be `extra_requirements`. See [https://docs.readthedocs.io/en/stable/config-file/v1.html#python-extra-requirements](https://docs.readthedocs.io/en/stable/config-file/v1.html#python-extra-requirements)